<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Line0-W</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Addr</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Len</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Bytes(Bytes)</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>WriteOnce</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QPushButton" name="pushButtonL0WOnce">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3" colspan="2">
        <widget class="QPushButton" name="pushButtonL0WClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="5">
        <widget class="QPlainTextEdit" name="plainTextEditL0WData">
         <property name="plainText">
          <string>1234</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>WriteContinue</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="pushButtonL0WContinue">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QLineEdit" name="lineEditL0WRate"/>
       </item>
       <item row="5" column="4">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Rate(MB/s)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QLineEdit" name="lineEditL0WLen">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="4">
        <widget class="QLineEdit" name="lineEditL0WAddr">
         <property name="text">
          <string>0x00001000</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="4">
        <widget class="QLineEdit" name="lineEditL0WBytes"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>Line0-R</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEditL0RLen">
         <property name="text">
          <string>200</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="4">
        <widget class="QPlainTextEdit" name="plainTextEditL0RData"/>
       </item>
       <item row="10" column="1">
        <widget class="QPushButton" name="pushButtonL0RContinue">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Addr</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Bytes(Bytes)</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>ReadOnce</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEditL0RAddr">
         <property name="text">
          <string>0x00001000</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Len</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>ReadContinue</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEditL0RBytes"/>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButtonL0ROnce">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QPushButton" name="pushButtonL0RClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QLineEdit" name="lineEditL0RRate"/>
       </item>
       <item row="10" column="3">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>(MB/s)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
